### ISSUE TITLE
Clean up unpublished collections that aren't in use

### ISSUE THREAD

**Issue by hackmastera** [pulibrary/pomegranate#709]
Are all the unpublished collections under active development or testing? Are there any that people messed around with but don't plan to go back to? If so it would be helpful to delete those.

**Comment by kelea99** [comment]
@hackmastera , I cannot delete this collection, as it says it doesn't exist. can you help? thanks! https://dpul.princeton.edu/dpul.princeton.edu%2Fslavic. I actually cannot delete any, as it is still giving me the 405 error

**Comment by hackmastera** [comment]
@kelea99 I will take a look.

**Comment by hackmastera** [comment]
@kelea99 I just took a look at this. I do have a memory that the "Late Soviet Posters" collection had a bad slug in figgy, and that's probably where this url came from (`https://dpul.princeton.edu/dpul.princeton.edu%2Fslavic`). I'll go ahead and delete it from the console.

Can you give me an example of another one you wanted to delete, for which you were getting the 405?

**Comment by kelea99** [comment]
@hackmastera, sure! Bibliotheca Cicognara (unpublished) was one of them, but let me try again. I looked and it was still there.  standby

**Comment by kelea99** [comment]
Same thing happened just now, when trying to delete the collection, @hackmastera. : 
<img width="1440" alt="Screen Shot 2020-03-31 at 2 57 20 PM" src="https://user-images.githubusercontent.com/45043014/78064605-0075db80-7360-11ea-8954-9e54905f5b59.png">

**Comment by hackmastera** [comment]
thanks!

**Comment by hackmastera** [comment]
I spent a bit of time looking into this and I haven't yet found the reason for the permission error. If you'd like to just give a list of the ones that should be deleted I can do them from the console. That would at least get rid of the collections we aren't using.

**Comment by hackmastera** [comment]
I also get the 405 through the UI. Checked permissions set via ability and we should be able to destroy this exhibit given our admin status.

```
> ability = Ability.new(ah)
> ability.can? :destroy, e
=> true
```
